src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -125,7 +121,7 @@ int GY50::readL3G4200DRegister(uint8_t registerAddress)
     mRuntime.i2cEndTransmission();
     mRuntime.i2cRequestFrom(kGyroAddress, 1);
 
-    return mRuntime.i2cAvailable() != 0 ? mRuntime.i2cRead() : 0;
+    return mRuntime.i2cAvailable() ? mRuntime.i2cRead() : 0;
```

Comment:
:warning: **readability-implicit-bool-conversion** :warning:
implicit conversion 'int' -> bool
```suggestion
    return  != 0```